guides/creating-a-custom-form-field-control.md

```diff
@@ -307,22 +307,14 @@ make up our component.
 
 ```ts
 @Input()
-get disabled() {
-  return this._disabled;
-}
-set disabled(dis) {
-  this._disabled = coerceBooleanProperty(dis);
+get disabled(): boolean { return this._disabled; }
+set disabled(value: boolean) {
+  this._disabled = coerceBooleanProperty(value);
+  this._disabled ? this.parts.disable() : this.parts.enable();
```

Comment:
Should this be added to the final result in `form-field-custom-control-example.ts` as well?

Reply:
@crisbeto Yes, this PR amends the guide and example to match. Thanks

Reply:
Ah, I must've missed it the first time around.